src/reason-parser/reason_pprint_ast.ml

```diff
@@ -4675,7 +4675,11 @@ let printer = object(self:'self)
           source_map ~loc:ct.ptyp_loc
             (self#non_arrowed_non_simple_core_type ct)
         in
-        (argsList@[formatJustTheTypeConstraint typeLayout], e)
+        ([makeList
+           ~break:IfNeed
+           ~indent:(settings.space * settings.indentWrappedPatternArgs)
```

Comment:
oh man, settings.space? Have we been using this consistently? cc @IwanKaramazow

Reply:
I used the same setting that `wrapCurriedFunctionBinding` employs, given that's where it all trickled down before.

Reply:
`settings.space` exists and the space setting is used only in function bindings I think. It's pretty obscure, since it defaults to the default `2` here.